pkg/models/quotas.go

```diff
@@ -18,9 +18,10 @@ type Quota struct {
 }
 
 type QuotaScope struct {
-	Name         string
-	Target       string
-	DefaultLimit int64
+	Name             string
+	Target           string
+	TargetConditions string
```

Comment:
I considered making this `[]string` such that each condition (if there were multiple) would be more clear to discern.

Reply:
I think that could be equally as messy as it would be somewhat clearer to discern, however. It would also make https://github.com/grafana/grafana/pull/32519#discussion_r604681078 unrealistic as an approach - if that approach is adjudged to be appropriate.

Reply:
This should be fine. Would that be needed in the future we can change it